CHANGELOG.md

```diff
@@ -17,8 +16,17 @@ BUG FIXES:
 
  * vault: Fix renewal time to be 1/2 lease duration with jitter [[GH-5479](https://github.com/hashicorp/nomad/issues/5479)]
  * metrics: Fixed stale metrics [[GH-5540](https://github.com/hashicorp/nomad/issues/5540)]
- * logging: Fix respawning logmon during task restarts when the logmon process
-   has been terminated [[GH-5577](https://github.com/hashicorp/nomad/pull/5577)]
+
+## 0.9.1 (April 23, 2019)
```

Comment:
0.9.1 hasn't been released yet either

Reply:
@notnoop I copied the release date from the release branch but yeah - I'll flag it as unreleased for now